config/src/test/java/org/axonframework/config/EventProcessingModuleTest.java

```diff
@@ -1385,6 +1385,51 @@ void interceptorsOnDeadLetterProcessorShouldBePresent(
         assertEquals(3, interceptors.size());
     }
 
+    @Test
+    void registerDeadLetterQueueProviderConstructsDeadLetteringEventHandlerInvoker(
```

Comment:
Nit: Can we add a test that a more specific DLQ registration overrides this default DeadLetterQueueProvider?

Reply:
added